src/freenas/usr/local/libexec/freenas-debug/zfs/zfs.sh

```diff
@@ -40,8 +40,8 @@ zfs_func()
 	zpool status
 	section_footer
 
-	section_header "ZFS Pools History"
-	zpool history
+	section_header "ZFS Pools History - excepting replication"
+	zpool history |  egrep -v "zfs.*(snapshot|destroy|recv).*"
```

Comment:
I totally understand why the replication info should be excluded but on the other hand this information is useful.
Maybe another section with full ZFS history? I don't have strong opinion on this one.